.github/workflows/keep-prisma-dependencies-updated.yaml

```diff
@@ -59,7 +59,12 @@ jobs:
   report-to-slack-failure:
     if: github.repository_owner == 'prisma' && failure()
     runs-on: ubuntu-latest
-    needs: [check-for-update-latest,  check-for-update-dev,  check-for-update-patch-dev]
+    needs:
+      [
+        check-for-update-latest,
+        check-for-update-dev,
+        check-for-update-patch-dev,
+      ]
```

Comment:
don't do this, it should be either ['',''] or 
needs:
  - 1
  - 2

Reply:
This was prettier or something. Reverting.

Reply:
```suggestion
    needs:
      [check-for-update-latest, check-for-update-dev, check-for-update-patch-dev]
```